lib/rules/no-function-prototype-bind.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * @author Yosuke Ota <https://github.com/ota-meshi>
+ * See LICENSE file in root directory for full license.
+ */
+"use strict"
+
+const {
+    definePrototypeMethodHandler,
+} = require("../util/define-prototype-method-handler")
+
+module.exports = {
+    meta: {
+        docs: {
+            description: "disallow the `Function.prototype.bind` method.",
+            category: "ES5",
```

Comment:
If you release it in a minor version, we need to change it to undefined so that it doesn't affect the ruleset.
Let me know if I need to change.